packs/slack/sensors/slack_sensor.yaml

```diff
@@ -20,3 +20,17 @@
             type: "integer"
           timestamp_raw:
             type: "string"
+    -
+      name: "message_channel_join"
```

Comment:
Maybe just `join` or perhaps `channel.join` (I find the message part confusing)?